benchmark/browser/index.js

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import ReactDOM from 'react-dom';
+import * as ReactDOM from 'react-dom';
```

Comment:
A bonus refactoring, to match the docs: https://reactjs.org/docs/react-dom.html.